### PR TITLE
PDA-181 navigation fix

### PR DESCRIPTION
**Pull request by rimselis**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-jx** [comment]
@rimselis: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 2cb19d7de37409225744d2c20ff3bc7ff99e550f | [link](https://raw.githubusercontent.com/entando/devops-results/logs/jenkins-x/logs/entando/entando-process-driven-plugin/PR-111/2.log) | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by gReis89** [comment]
when accessing one of the pages, and going back to individual widgets, the font size of titles are huge:

Screen Shot 2020-04-23 at 17 14 34

File: widgets/src/api/pda/tasks.js
Comment (gReis89) [inline widgets/src/api/pda/tasks.js:12]: since we are refactoring that, won't be better to merge both objects into a single `params`?

File: widgets/src/api/pda/tasks.js
Comment (gReis89) [inline widgets/src/api/pda/tasks.js:70]: it looks more an implementation of an existing API rather than a new one, shouldn't it be in the component layer?

File: widgets/src/components/App/App.js
Comment (gReis89) [inline widgets/src/components/App/App.js]: can you move the `App` folder one level up to `src` folder? It is something I wanted to do for some time.

**Comment by rimselis** [comment]
> when accessing one of the pages, and going back to individual widgets, the font size of titles are huge:

It seems to remain even when moving from widget to widget. Strange browser behaviour

Might be related to the order the css is loaded as there are rules for Typography twice

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/common/DropDownButton.js
Comment (gReis89) [inline widgets/src/components/common/DropDownButton.js:49]: on Figma designs, button doesn't have elevation, why have u removed `disableElevation` here?

Comment (rimselis) [reply widgets/src/components/common/DropDownButton.js:49]: `ButtonGroup` does not support `disableElevation` (https://material-ui.com/api/button-group/), that leads to an error in console as it's passed down to DOM element.

**Comment by entando-service** [comment]
All is well

File: widgets/src/api/pda/tasks.js
Comment (rimselis) [reply widgets/src/api/pda/tasks.js:70]: I would like to keep it separated from component level in case we need it elsewhere or when we look into global state/sub-pub.

**Comment by entando-service** [comment]
All is well

File: widgets/src/api/pda/tasks.js
Comment (gReis89) [reply widgets/src/api/pda/tasks.js:70]: ok, makes sense!

File: widgets/src/App/App.js
Comment (gReis89) [inline widgets/src/App/App.js]:
```
Casing of app/Menu does not match the underlying filesystem
```

it should be `'App/Menu'`

**Comment by gReis89** [comment]
can u change the `knowledgeSource` from `taskCommentsConfigs` object in `mocks/app-builder/pages.js` to point to `KieStaging`? all mocks are pointing to KieStaging but the Comments one that is pointing to KieDev.

File: widgets/src/App/App.js
Comment (rimselis) [reply widgets/src/App/App.js]: it's the only capitalized folder in that level, so it seemed out of place

Comment (gReis89) [reply widgets/src/App/App.js]: I agree, but the folder is still called `App`, can we rename it to `app`?

Comment (rimselis) [reply widgets/src/App/App.js]: seems like folder name change did not commit...

**Comment by entando-service** [comment]
All is well

**Review by gReis89 — APPROVED** [review]
looks good!

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/111#pullrequestreview-401874030" title="Approved">gReis89</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
